src/openai/lib/streaming/_assistants.py

```diff
@@ -906,11 +906,11 @@ def accumulate_run_step(
             merged = accumulate_delta(
                 cast(
                     "dict[object, object]",
-                    snapshot.model_dump(exclude_unset=True),
+                    snapshot.model_dump(exclude_unset=True, warnings=False),
```

Comment:
if PYDANTIC_V2 then only send warnings as False

Reply:
Did this break on pydantic v1 for you?

Reply:
ah yeah it did, not sure how our tests didn't catch that, will push a fix

Reply:
this will be fixed in the next release https://github.com/openai/openai-python/pull/1735